pymodule.py

```diff
@@ -133,7 +132,8 @@ def dipole_routine(dsrg_method, reference):
 
             # Compute the energy
             if is_multi_state and ms_dsrg_algorithm == "SA_SUB":
-                state_energies_list = active_space_solver.compute_contracted_energy(ints_dressed)
+                sa_sub_max_rdm = 2 # TODO: is the 2 here correct?
```

Comment:
It depends. This should be 3 if do_hbar3 is true. However it is not supported in compute_contracted_energy yet.